.docker/images/nginx/tests/src/AcsfRedirectsTest.php

```diff
@@ -42,6 +42,7 @@ public function providerRedirectPath() {
   public function testRedirectPath($path) {
     $headers = \get_curl_headers($path);
     $this->assertEquals(301, $headers['Status']);
+    $this->assertTrue(strpos($headers['Location'], 'https://') > -1);
```

Comment:
how about

```suggestion
    $this->assertTrue(preg_match('#^https://#', $headers['Location']));
```

I think this is easier to read.

Reply:
`assertTrue` looks to do a `===` and preg_match returns a match; so `1 !== TRUE` and this now fails - I'll update it though so we can use the starts with regex.

Reply:
For reference.

```
1) GovCMSTests\AcsfRedirectsTest::testRedirectPath with data set #0 ('/sites/g/files/net12409/theme...st.jpg')
Failed asserting that 1 is true.
```